code/common/finspace.q

```diff
@@ -48,4 +49,9 @@ eopdatacleanup:{[dict]
     hclose each distinct exec w from .sub.SUBSCRIPTIONS;
     // function to parse icounts dict and remove all data after a given index for RDB and WDB's 
     {[t;ind]delete from t where i >= ind}'[key dict;first each value dict];
- }
\ No newline at end of file
+ }
+//set rdbready to true after signal received from the old rdb, that new processes are running and ready to take over at start of new period
+newrdbup:{[]
+        .lg.o[`newrdbup;"received signal from next period rdb, setting rdbready to true"]
```

Comment:
indenting doesn't match the rest of the file

Reply:
indentation seems to be inconsistent in this one, i've adjusted it so that its consistent with the notifyhdb and createchangeset functions above it